Unite.Web/Client/src/domain/donors/api/index.js

```diff
@@ -1,15 +1,15 @@
-import settings from "@/settings";
-import DomainApi from "@/domain/_shared/api/domain-api";
+import settings from '@/settings';
```

Comment:
Use double quotes "" wherever and whenever possible and single quotes '' for interpolation between "" e.g. "Values is 'hello'";